src/mono/netcore/System.Private.CoreLib/src/System/Reflection/Emit/AssemblyBuilder.Mono.cs

```diff
@@ -419,13 +419,13 @@ public void SetCustomAttribute(ConstructorInfo con, byte[] binaryAttribute)
             SetCustomAttribute(new CustomAttributeBuilder(con, binaryAttribute));
         }
 
-        private Exception not_supported()
+        private static Exception not_supported()
         {
             // Strange message but this is what MS.NET prints...
             return new NotSupportedException("The invoked member is not supported in a dynamic module.");
         }
 
-        private string create_assembly_version(string version)
+        private static string create_assembly_version(string version)
```

Comment:
This seems to be unused, I think we can just get rid of it?
```
ryan@kenshin:~/git/runtime$ git grep create_assembly_version
src/mono/netcore/System.Private.CoreLib/src/System/Reflection/Emit/AssemblyBuilder.Mono.cs:        private string create_assembly_version(string version)
```

Reply:
Removed